CefSharp.Wpf.Example/Views/BrowserTabView.xaml

```diff
@@ -168,12 +168,15 @@
                     </Rectangle.Fill>
                 </Rectangle>
 
+                    <ContentControl Content="{Binding WebBrowser}"
+                                Opacity="{Binding ElementName=opacitySlider, Path=Value}" >
+                    <!--
                 <cefSharp:ChromiumWebBrowser x:Name="browser"
```

Comment:
> it seems the underlying control should probably manage that.

Which control were you thinking? I'm sure we can come up with something, just have to ponder for a bit....